### DISCUSSION TITLE
Is HDD-fragmentation an issue?

### DISCUSSION THREAD

**Discussion by Muffexx** [Chia-Network/chia-blockchain#2819] [Q&A] [answered]
Hello there,

I am currently plotting with several SSDs to the same HDD. When writing 4+ files simultaneously on a NTFS-formatted HDD will/might there be issues with fragmentation that can affect farming?

regards

**Comment by cccat6** [comment, marked as answer]
Not an issue, it doesn't matter.

**Reply by Muffexx** [reply]
Thanks for your answer.

Do you think it doesn't even matter if the files are almost all heavily interleaved?

regards

**Reply by cccat6** [reply]
All the read requests of farming of plot files with buffer 4096 or 512. There is all random access. No sequential read necessary. Check for proof is also a light task for the drive. So no difference for that.

**Reply by Muffexx** [reply]
This is very relieving, as i had already started rewriting some plots on the next HDD.
Thank you very much.
Case closed.
